src/babies/babies.controller.ts

```diff
@@ -28,13 +30,16 @@ export class BabiesController {
   }
 
   @Get('/:id')
-  getBaby(@Param('id', new ParseUUIDPipe()) id: string) {
-    return this.babiesService.getBaby(id);
+  getBaby(@Request() request, @Param('id', new ParseUUIDPipe()) id: string) {
```

Comment:
The baby identifier is an `id` or `uuid`? I bet that it's a `uuid`, right? Since you are typing it as a string.... just asking because this question crossed my mind while reviewing, does it make sense to rename it as `uuid` in the parameter or do you want to keep the `id` as `uuid` approach?

Reply:
it's a `uuid`, not sure if I see the need to rename it though... it's an identifier, of which kind, I'm not sure it matters here. what do you think?

Reply:
I think having the `id` typed as a `string` is enough to understand that it may be a `uuid` but as you said ... it's an identifier and we probably don't care about which type it is in this context